packages/@react-spectrum/button/intl/en-US.json

```diff
@@ -1,3 +1,3 @@
 {
-  "loading": "Loading…"
+  "pending": "pending"
```

Comment:
Shouldn't pending be capitalized?

Reply:
I'm not sure it really matters
https://stackoverflow.com/questions/49930658/should-aria-labels-be-capitalized#:~:text=It%20doesn't%20matter%20if,be%20read%20as%20separate%20characters.